cudamapper/src/cudamapper_utils.hpp

```diff
@@ -28,28 +28,9 @@ namespace claraparabricks
 namespace genomeworks
 {
 
-namespace io
```

Comment:
I can't comment on lines which were not edited. I would also remove `#include <mutex>` above

Reply:
`#include <claraparabricks/genomeworks/utils/allocator.hpp>` also looks suspicious (probably not part of this PR, but we can fix it while we're here)

Reply:
Fixed. Removed mutex and allocator